frontend/src/pages/TagsPage/TagsPage.js

```diff
@@ -30,9 +31,13 @@ const TagsPage = () => {
   if (tags.isError) return <h1>Something went wrong!</h1>
   if (tags.isLoading) return <h1>Loading...</h1>
   return (
+    <>
     <div className="bg-white h-full overflow-y-scroll">
       <Table columns={columns} data={tags?.data?.data?.data?.tags} />
+      <Link to="/admin/tags/create">Add</Link>
```

Comment:
This is okay for now but will need to be reused on every single show all page. Should really be it's own component but I'll move this to a new ticket